lib/ngx/balancer_stream.lua

```diff
@@ -0,0 +1,82 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
I think `ngx.balancer.stream` is a better module name than `ngx.balancer_stream` :)